Hunting Queries/CustomLogs/AWSBucketAPILogs-S3BucketDataTransferTimeSeriesAnomaly.yaml

```diff
@@ -1,10 +1,12 @@
 id: 0ef8dee1-eb94-44c8-b59b-2eb096a4b983
 name: S3 Bucket outbound Data transfer anomaly
 description: |
```

Comment:
needs single quotes around just in case characters get updated

Reply:
updated